commands/zcount.md

```diff
@@ -11,11 +11,12 @@ Note: the command has a complexity of just O(log(N)) because it uses elements ra
 @integer-reply: the number of elements in the specified score range.
 
 @examples
+- `-inf +inf` : 'inf' stands for infinity, meaning from least negative infinity to maximum positive infinity.
```

Comment:
I don't think this explanation should be under "Examples". It's better to explain it in the main documentation section. We should also explain exclusive ranges with `(`.

We already have "The min and max arguments have the same semantic as described for ZRANGEBYSCORE.", but ZRANGEBYSCORE is deprecated and replaced by ZRANGE with the BYSCORE argument. So instead, I think we should replace "The min and max arguments have the same semantic as described for ZRANGEBYSCORE." with the same explanation that we have in the ZRANGE docs:

> **start** and **stop** can be **-inf** and **+inf**, denoting the negative and positive infinities, respectively. This means that you are not required to know the highest or lowest score in the sorted set to get all elements from or up to a certain score.
>
> By default, the score intervals specified by **start** and **stop** are closed (inclusive). It is possible to specify an open interval (exclusive) by prefixing the score with the character `(`.